tests/aws/services/apigateway/test_apigateway_sqs.py

```diff
@@ -19,15 +19,20 @@
 
 @markers.aws.unknown
 def test_api_gateway_sqs_integration(aws_client):
-    # create target SQS stream
+    # create target SQS stream'
     queue_name = f"queue-{short_uid()}"
-    resource_util.create_sqs_queue(queue_name)
+    queue_arn = arns.sqs_queue_arn(
+        queue_name, region_name=TEST_AWS_REGION_NAME, account_id=TEST_AWS_ACCOUNT_ID
+    )
+    resource_util.create_sqs_queue(
+        queue_name, account_id=TEST_AWS_ACCOUNT_ID, region_name=TEST_AWS_REGION_NAME
+    )
```

Comment:
this should use a fixture instead (`sqs_create_queue`), so we actually could remove the util in `aws/resources`.